modules/genomics_cromwell/README.md

```diff
@@ -0,0 +1,35 @@
+# RAD Lab Genomics-Cromwell Module
+
+## GCP Products/Services 
+
+* Life Sciences API
+* Cloud Compute
+* CloudSQL
+* Cloud Storage
+* Virtual Private Cloud (VPC)
+
+## Module Overview 
+Cromwell is a Workflow Management System geared towards scientific workflows. Cromwell is open sourced under the BSD 3-Clause license. Checkout Cromwell documentation at https://cromwell.readthedocs.io/
+
+The RAD Lab Genomics module deploys Cromwell server along with a a CloudSQL instance and adds a firewall rule enabling access to the server through IAP Tunnel.
+
+This setup allows you to securely access the cromwell server bu submitting jobs from your device through a secure tunnel without the need to add a public IP to your cromwell server and also access the web UI using your browser through that tunnel.
```

Comment:
Does this work in Cloud Shell as well?  Submitting the jobs won't be problem, but I'm not a 100% sure the Web Preview will work.  Given that CS is the main platform, it would be great if you could test this.  Perhaps it's possible changing the port to >30000, as anything lower won't work.

Reply:
This works with Cloud Shell as well. The reason I picked 8080 is it is the default port for web preview, so after starting the IAP tunnel the user can simply click on the web-preview and directly access the server Web UI similar to what AppEngine does